aws/apigateway-aurora-lambda/functions/employee-office/join/index.js

```diff
@@ -0,0 +1,28 @@
+import { getDB } from '@models';
+import { success, failure, logHandler, getSystemId } from '@utils';
+export const handler = async (event, context, callback) =>
+  logHandler(event, async () => {
+    try {
+      const { employeeId, officeId } = JSON.parse(event.body);
+      if (!getSystemId(event)) {
+        throw new Error('Request Id Missing!');
+      }
+      console.log({ employeeId, officeId });
+      const employee = await getDB().employees.findOne({ where: { id: employeeId }, raw: true });
+      const office = await getDB().offices.findOne({ where: { id: officeId }, raw: true });
+      const res = await getDB().employee_office.upsert({
```

Comment:
```
const { employee, officeEmployees } = getDB()
```


also let's use camelcase in the models/index when we're setting the keys for the db object.